src/tokenserver/auth/browserid.rs

```diff
@@ -65,9 +65,8 @@ impl VerifyToken for RemoteVerifier {
             })
             .send()
             .await
-            .and_then(Response::error_for_status)
             .map_err(|e| {
-                if e.is_connect() || e.is_status() {
+                if e.is_connect() {
```

Comment:
I had to revert this back because it wasn't handling 2XX errors other than 200 correctly (the old Tokenserver returns a 503 to the client if FxA returns anything other than a 200, not just anything other than a 2XX)